src/events/EventsPageBanner.styles.ts

```diff
@@ -49,7 +49,7 @@ export const Title = styled.h1`
 
 export const Subtitle = styled.p`
   font-size: 1rem;
-  margin-bottom: 0.5em;
+  margin-bottom: 1.5em;
```

Comment:
Can we take this from the `space` module instead? Should be located in `src/theme/space.ts`.

Reply:
Additionally, using the `Typography` component should make it easier, and you won't need to have some of these stylings manually applied.